app/src/main/java/com/randomappsinc/studentpicker/choosing/NameChoosingHistoryManager.java

```diff
@@ -0,0 +1,46 @@
+package com.randomappsinc.studentpicker.choosing;
+
+import android.content.Context;
+
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.randomappsinc.studentpicker.R;
+import com.randomappsinc.studentpicker.models.ListInfo;
+import com.randomappsinc.studentpicker.utils.NameUtils;
+import com.randomappsinc.studentpicker.utils.UIUtils;
+
+public class NameChoosingHistoryManager {
+
+    private ListInfo listInfo;
+    private MaterialDialog dialog;
+
+    NameChoosingHistoryManager(ListInfo listInfo, Context context) {
+        this.listInfo = listInfo;
+
+        dialog = new MaterialDialog.Builder(context)
+                .title(R.string.chosen_names_history)
+                .positiveText(android.R.string.yes)
```

Comment:
I think there's an R.string.okay we can use. Otherwise, let's define it in the common strings.xml file and use it. Using the Android defaults isn't super readable IMHO (yes is actually "Ok") and it leaves us vulnerable to SDK changes.